.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+### Type of change
+
+- [ ] bug fix
+- [ ] new feature
+- [ ] documentation
+- [ ] other (describe)
+
+### Scope of changes
+
+Briefly describe the scope of changes. Include links to ShortCut or Jira ticket if applicable.
+
+### Acceptance criteria
+
+Describe how reviewers can test this change to be sure that it works correctly. Add a checklist if possible.
+
+### Author checklist
+
+- [ ] I have manually tested the change and/or added automation in the form of unit tests or integration tests
+- [ ] I have added migrations for any changes to the database
+- [ ] My migrations can be run before the code is deployed without causing any issues
+- [ ] Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
+- [ ] I have moved the associated ShortCut or Jira ticket to "Ready for Review"
+
+### Reviewer(s) checklist
+
+- [ ] The code in the PR has been reviewed for architectural concerns and correctness
+- [ ] Migrations work without causing API downtime
+- [ ] The acceptance criteria have been reviewed and tested
```

Comment:
Maybe we should also add a checklist item for QA? e.g. 
- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.